examples/nextjs-spa/styles/globals.css

```diff
@@ -0,0 +1,26 @@
+html,
+body {
+  padding: 0;
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
+    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+@media (prefers-color-scheme: dark) {
+  html {
+    color-scheme: dark;
+  }
+  body {
+    color: #ededed;
+    background: #2b2b2b;
+  }
+}
```

Comment:
Add to this file the container css to place everything rendered in the center of the page (some padding top and bottom) so that your login / registration etc cards are always centered.